lib/features/login/presentation/pages/login_options.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+
+import '../widgets/long_button.dart';
+import '../../../../utilities/constants.dart';
+
+class LoginOptions extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        child: Padding(
+          padding: EdgeInsets.only(top: 65.0),
+          child: Align(
+            alignment: Alignment.topCenter,
+            child: Container(
+              width: 236,
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.center,
+                children: <Widget>[
+                  Container(
+                    width: 156.0,
+                    height: 156.0,
+                    decoration: BoxDecoration(
+                      shape: BoxShape.circle,
+                      color: Color(0xFFC4C4C4),
```

Comment:
I'm wondering if we should start putting these colours in a file that we access so they have more readable names like "login_button_colour". Would this be in models?

Reply:
Ya that makes sense. I'm putting it into the constants file. Later on when we refactor for the full fledge Clean Archtecture we can think of the best place to put it in.

Change on [b36bb78](https://github.com/rishFilet/swipe_n_swap/pull/2/commits/b36bb78e19d1e18f4ac9596653df6ec7327331e8)